pkg/roachprod/promhelperclient/client_test.go

```diff
@@ -52,18 +54,27 @@ func TestUpdatePrometheusTargets(t *testing.T) {
 		require.Equal(t, "request failed with status 400 and error failed", err.Error())
 	})
 	t.Run("UpdatePrometheusTargets succeeds", func(t *testing.T) {
+		nodeInfos := map[int]string{1: "n1", 3: "n3"}
 		c.httpPut = func(ctx context.Context, url string, h *http.Header, body io.Reader) (
 			resp *http.Response, err error) {
 			require.Equal(t, getUrl(promUrl, "c1"), url)
 			ir, err := getInstanceConfigRequest(io.NopCloser(body))
 			require.Nil(t, err)
-			// TODO (bhaskar): check for the correct yaml
 			require.NotNil(t, ir.Config)
+			configs := make([]*CCParams, 0)
+			require.Nil(t, yaml.UnmarshalStrict([]byte(ir.Config), &configs))
+			require.Equal(t, 2, len(configs))
```

Comment:
Nit: `require.Len(t, configs, 2)` would make for better error messages (since it would include the actual configs).